crmsh/sh.py

```diff
@@ -0,0 +1,321 @@
+"""Run shell commands.
+
+This module provides various methods to run shell commands, on both local and remote hosts, as current or another user.
+There many variant of the methods to allow fine-gain control of parameter passing and error handling.
+
+3 different implementations are provided:
+
+1. LocalShell allows to run command on local host as various users. It is the most feature-rich one, allowing
+   interactive I/O from/to the terminal.
+2. SshShell allows to run command on both local and remote hosts. When running on a remote host, it creates a direct
```

Comment:
I think `SshShell` should be renamed as `SSHShell`, since `SSH` is a specific word instead of an English word